src/Task/Asset/ProcessAssetTask.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Synolia\SyliusAkeneoPlugin\Task\Asset;
+
+use BluePsyduck\SymfonyProcessManager\ProcessManagerInterface;
+use Doctrine\ORM\EntityManagerInterface;
+use Psr\Log\LoggerInterface;
+use Synolia\SyliusAkeneoPlugin\Logger\Messages;
+use Synolia\SyliusAkeneoPlugin\Payload\Asset\AssetPayload;
+use Synolia\SyliusAkeneoPlugin\Payload\PipelinePayloadInterface;
+use Synolia\SyliusAkeneoPlugin\Task\AbstractProcessTask;
+
+final class ProcessAssetTask extends AbstractProcessTask
+{
+    public function __construct(
+        EntityManagerInterface $entityManager,
+        LoggerInterface $akeneoLogger,
+        ProcessManagerInterface $processManager,
+        BatchAssetTask $task,
+        string $projectDir
+    ) {
+        parent::__construct(
```

Comment:
useless construct definition

Reply:
thought the same when re-reading, but here we pass the BatchAssetTask as $task and we call the parent which accepts BatchTaskInterface